plugins/mcreator-core/procedures/item_nbt_copy.json

```diff
@@ -17,6 +17,10 @@
   "colour": 350,
   "mcreator": {
     "toolbox_id": "itemmanagement",
+    "toolbox_init": [
+      "~<value name=\"a\"><block type=\"mcitem_all\"><field name=\"value\"></field></block></value>",

```

Comment:
Maybe make this one a common setup, not just for tests (in another PR?)?

Reply:
Common setup?

Reply:
Remove the <kbd>~</kbd> in the beginning so that these are used in Blockly panels too.

Reply:
Hmm, copy usually expects provided itemstack or itemstack from GUI so mcitem_all is not really the expected input. It was intentionally left out blank as it is very broad operator